Whoaverse/Whoaverse/Scripts/whoaverse.js

```diff
@@ -204,7 +204,119 @@ $(document).ready(function () {
             }
         });
     });
-
+    
+    var key = {};
+
+    // List of the inputs the user use when creating a new discussion
+    var submitInputs = {
+      post: [
+        '#Title',
+        '#Subverse',
+        '#MessageContent'
+      ],
+      reply: [
+        '#CommentContent'
+      ]
+    };
+    
+    document.onkeydown = function(evt) {
+      var evt = evt || window.event;
+      var charCode = evt.keyCode || evt.which;
+      var charStr = String.fromCharCode(charCode);
+      
+      // Turn key numbers into readable string
+      switch(charCode) {
+        case 13:
+          key['Enter'] = true;
+          break;
+        case 17:
+          key['Ctrl'] = true;
+          break;
+        case 18:
+          key['Alt'] = true;
+          break;
+        case 27:
+          key['Esc'] = true;
+          break;
+        case 32:
+          key['Space'] = true;
+          break;
+        case 37:
+          key['Left'] = true;
+          break;
+        case 38:
+          key['Top'] = true;
+          break;
+        case 39:
+          key['Right'] = true;
+          break;
+        case 40:
+          key['Bottom'] = true;
+          break;
+        default:
+          key[charStr] = true;
+      }
+      
+      if(key.Enter && key.Enter === true && key.Ctrl && key.Ctrl === true) {
+        submitInputs.post.forEach(function(selector) {
+          if(document.querySelector(selector) == document.activeElement) {
+            document.querySelector('input.btn-whoaverse[type="submit"]').click();
+          }
+        });
+        
+        submitInputs.reply.forEach(function(selector) {
+          var inputs = document.querySelectorAll(selector);
+          
+          for(var i=0; i<inputs.length; i++) {
+            if(inputs[i] == document.activeElement) {
+              inputs[i].form.querySelector('#submitbutton').click();
+            }
+          }
+        });
+      }
+    };
+    
+    window.onkeyup = function(evt){
```

Comment:
> This is for multiple keypresses. Correct me if I'm wrong, but wouldn't using this code in the keydown listener dramatically simplify it    
    
Nay, this isn't for multiple keypresses. This code translates to this english:    
- uppon release of any key
- check which key was RELEASED
- set that released key status of "pressed" to `false` (aka not pressing that key any more)    
    
This is used because, otherwise, every time you'd press a key you'd set that key status as "pressed" (`true`) and since your code didn't knew about the "released key" you'd only be able to enter the `if(key.Enter && key.Enter === true && key.Ctrl && key.Ctrl === true) {` condition IF those were the first two keys you pressed. After that you'd never be able to enter that statement *ever again*.    
    
> if (charCode === 13 && evt.ctrlKey)    
    
Yes, this would, indeed, simplify the number of statements the interpreter had to read; but in turn it makes it harder for any other coder who wants to maintain this part of the code hard to understand. (plus, the less "machine code" you write, the more your future self will thank your past self - rule of thumb)    
    
The only thing I'd have done differently here, would be changing the `key['Enter']` for `key.Enter`; But that's just because `key` was originally an Object (`{}`) and using `[]` might mislead the newcomers to think that that is an array instead.